jack-core/src/com/rapleaf/jack/transaction/ITransactor.java

```diff
@@ -20,6 +20,10 @@
 
   void executeAsTransaction(IExecution<DB> execution);
 
+  boolean isMetricsTrackingEnabled();
+
+  void toggleMetricsTracking();
+
```

Comment:
Let's not expose these options to users for now. I am thinking that users should be allowed to enable or disable tracking only when constructing the transactor. Can you add a method (e.g. `disableMetricsTracking`) in the builder of the transactor?